packages/ens-deployments/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "@ensnode/shared-configs/tsconfig.ponder.json",
   "include": ["./**/*.ts"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "rootDir": "."
+  }
```

Comment:
This update was required to make TS happy again after including `@ensnode/ens-deployments` package on the list of dependencies for `@ensnode/ensnode-sdk` package.